CONTRIBUTING.md

```diff
@@ -69,3 +69,13 @@ flake8 .
 
 You can set the `PYGFX_LOG_LEVEL` environment variable to get more
 detailed log messages. Can be an int or any of the standard level names.
+
+
+## Attributes
+
+We distinguish four kinds of attributes:
+
+* Public attibutes for the user: should in most cases be ``@property``'s.
+* Private attributes: prefixed with "_" as usual.
```

Comment:
I also encountered `__update_buffer_color` which is not covered here

Reply:
Good point, we never use double underscores except for one place where it added. Will change that into a single underscore.